spec/cordial/contacts_spec.rb

```diff
@@ -27,3 +27,16 @@
     end
   end
 
+  describe '#unsubscribe' do
+    subject { described_class.unsubscribe(email: email) }
+
+    it 'has a correctly formatted request url' do
+      unsubscribe_url = 'https://api.cordial.io/v1/contacts/cordial@example.com/unsubscribe/email'
+      expect(subject.request.last_uri.to_s).to eq unsubscribe_url
+    end
+
+    it 'has the correct payload' do
+      expect(subject.request.raw_body).to eq 'mcID='
+    end
```

Comment:
So I think is going to fail now though :D

Reply:
Updated :) Thank you